docs/spark/how-to-guides/hdinsight-notebook-installation.md

```diff
@@ -0,0 +1,123 @@
+
+To provide the end-to-end integration, we had to add a Jupyter kernel along with necessary modifications to [Apache Livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic). Since these code changes have not yet been merged into the respective open source projects, to be able to get the out-of-box .NET for Apache Spark experience, you will have to patch components in your HDInsight Spark Cluster with the binaries we provide.
+
+To enable .NET for Apache Spark through the Jupyter Notebook Experience, you would have to follow a few manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and submit [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) on HDInsight Spark cluster.
+
+>**Disclaimer:** Please note that this is *experimental* and not supported by the HDInsight Spark team. *Use it at your own risk*. Please do remember that we are working hard to get these changes into the respective projects so you do not need these manual steps.
+
+## Pre-requisites:
+Create an [Azure HDInsight Spark](https://docs.microsoft.com/en-us/azure/hdinsight/spark/apache-spark-jupyter-spark-sql-use-portal#create-an-hdinsight-spark-cluster) cluster (by visiting the **[Azure Portal](https;//portal.azure.com)**, selecting **+ Create a Resource**, and creating a new Azure HDInsight cluster resource), select the **Overview** tab on your cluster. Ensure you have created a cluster with **Spark 2.4** and **HDI 4.0**.
+
+## Installation Procedure
+Open the [Azure portal](https://ms.portal.azure.com/#home). Select **HDInsight Spark cluster** you created in the previous step.
+### Step 1. Stop Livy Server
+1.1 From the portal, select **Overview**, and then select **Ambari home**. If prompted, enter the cluster login credentials for the cluster.
+
+<img src="../media/hdi-spark-notebooks/select-ambari.png" alt="StopLivyServerImage" width="800"/>
+
+1.2 Select **Spark2**, and then select **LIVY FOR SPARK2 SERVER**.
+
+<img src="../media/hdi-spark-notebooks/select-livyserver.png" alt="StopLivyServerImage" width="800"/>
+
+1.3 Select **hn0... host**, stop **Livy for Spark2 Server** if started. When prompted, click **OK** to proceed.
+
+- Select hn0 as shown below.
+<img src="../media/hdi-spark-notebooks/select-host.png" alt="StopLivyServerImage" width="800"/>
+
+- Stop Livy for Spark2 Server.
+<img src="../media/hdi-spark-notebooks/stop-server.png" alt="StopLivyServerImage" width="800"/>
+
+1.4 Please follow the same step for hn1... host.
+
+### Step 2. Submit HDInsight Script Action
+Return to your HDI cluster in the Azure Portal, and select **Script actions** from the options on the left. You'll submit one script action to deploy SparkDotnet REPL on HDInsight Spark cluster.
+
+Submit Script Action on both Head node and Worker node.
```

Comment:
```suggestion
You will submit the script action on both the head and worker nodes.
```

Reply:
Or, since the bulleted list below is great and outlines all of the necessary steps and info, could we remove this sentence (since users will see they need to run on the head and workers below anyways)?

Reply:
+1 for removing this line

Reply:
Removed.